src/testing/vortex/workload.zig

```diff
@@ -184,31 +206,69 @@ fn reconcile(result: Result, command: *const Command, model: *Model) !void {
                 try testing.expect(model.account_exists(transfer.debit_account_id));
                 try testing.expect(model.account_exists(transfer.credit_account_id));
             }
+
+            // Drop the oldest transfer IDs and add new ones.
+            for (0..@min(successful_transfer_ids.count(), model.latest_transfers.count)) |_| {
+                model.latest_transfers.retreat_tail();
```

Comment:
I think this is fine for now! If we need this in a couple of more places, we can add a helper to do this!